react-todo-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
```

Comment:
Kindly update the README file in a way that shows what this repo is all about. I suggest to use Microverse's README Template.